src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -162,8 +162,42 @@ private void appendOneNativeArgument(ExecutionContext context, object obj, char
                     arg = PSObject.ToStringParser(context, ParserOps.Current(null, list));
                 }
 
+#if UNIX
+                Collection<string> resolvedPaths = new Collection<string>();
+#endif
                 if (!String.IsNullOrEmpty(arg))
                 {
+                    // This implements globbing of arguments given to native tools on UNIX.
+                    // Conversely, Windows tools implement their own globbing,
```

Comment:
I would argue that we should provide some way to control it and not tide it to the platform. We can have different defaults on different platforms.

But the problem come with cross-platform tools (i.e. git). It would be pretty unfun to get inconsistent behavior on different platforms.